<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="data_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>841</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="request_edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>600</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>660</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> Добавить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="aspect_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="type_box">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>600</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="place_box">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="level_box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
